command.rkt

```diff
@@ -0,0 +1,40 @@
+#lang racket/base
+
+(require racket/contract)
+
+(provide (contract-out
+          [run-command (->* (string?) #:rest (listof (or/c string? path?))
+                            string?)]))
```

Comment:
You can use ellipses with the `->` contract combinator to specify rest arguments more easily.
```suggestion
          [run-command (-> string? (or/c string? path?) ... string?)]))
```